cache/network/mocklogger_test.go

```diff
@@ -0,0 +1,63 @@
+package network
```

Comment:
I do not see the usual testify comment at the top so I am guessing you write this manually. Just wondering why.

Reply:
This uses `testify`'s mocking, but I did not generate it automatically with `mockery` 🙂 I copied it over from another step package because the IDE recognized that there are a couple of existing implementations of the logger interface.